pkg/services/sentry/config_test.go

```diff
@@ -7,26 +7,49 @@ import (
 )
 
 func Test_Sentry_ReadFiles(t *testing.T) {
-	configWithSentryDisabled := NewConfig()
-	configWithSentryDisabled.Enabled = false
-	configWithSentryDisabled.KeyFile = "a file that does not exists so that to show that we are not performing file reading"
+	type fields struct {
+		config *Config
+	}
 	tests := []struct {
-		name    string
-		config  *Config
-		wantErr bool
+		name     string
+		fields   fields
+		modifyFn func(config *Config)
+		wantErr  bool
 	}{
 		{
-			name:    "Do not read file when sentry disabled",
+			name: "Do not read file when sentry disabled",
+			fields: fields{
+				config: NewConfig(),
+			},
+			modifyFn: func(config *Config) {
+				config.Enabled = false
+				config.KeyFile = "a file that does not exists so that to show that we are not performing file reading"
+			},
 			wantErr: false,
-			config:  configWithSentryDisabled,
+		},
+		{
+			name: "Should return an error with non-existent KeyFile and sentry enabled.",
+			fields: fields{
+				config: NewConfig(),
+			},
+			modifyFn: func(config *Config) {
+				config.Enabled = true
+				config.KeyFile = "a file that does not exists to show that we will perform a file reading with sentry enabled and return error"
+			},
+			wantErr: true,
 		},
 	}
 
 	RegisterTestingT(t)
+
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			err := tt.config.ReadFiles()
-			Expect(tt.wantErr).To(Equal(err != nil))
+			config := tt.fields.config
```

Comment:
I believe the `config` can be removed from the fields and this can be changed to:
```suggestion
			config := NewConfig()
```

Reply:
Nice find, thanks @ziccardi :)